src/config/defaults_cat_general.go

```diff
@@ -28,18 +28,14 @@ var generalRequiredByOriginEngines = []engines.Name{
 	engines.GOOGLE,
 }
 
-var generalPreferredEngines = []engines.Name{
-	engines.ETOOLS, // Not in ByOrigin because it only gives 10 results across a lot of engines that it scrapes from.
```

Comment:
Etools gives very few results which we already get from other engines, doesn't work most of the time (because of bot detection / ip rate limit) so there's no need to wait for it in general category.